ssv/spectest/tests/partialsigcontainer/duplicate.go

```diff
@@ -0,0 +1,36 @@
+package partialsigcontainer
+
+import (
+	"github.com/bloxapp/ssv-spec/qbft"
+	"github.com/bloxapp/ssv-spec/ssv/spectest/tests"
+	"github.com/bloxapp/ssv-spec/types"
+	"github.com/bloxapp/ssv-spec/types/testingutils"
+)
+
+func Duplicate() tests.SpecTest {
+
+	// Create a test key set
+	ks := testingutils.Testing4SharesSet()
+
+	// Create PartialSignatureMessage for testing
+	msg1 := testingutils.PostConsensusAttestationMsg(ks.Shares[1], 1, qbft.FirstHeight)
+	msg2 := testingutils.PostConsensusAttestationMsg(ks.Shares[1], 1, qbft.FirstHeight)
+	msg3 := testingutils.PostConsensusAttestationMsg(ks.Shares[2], 2, qbft.FirstHeight)
+	msgs := []*types.PartialSignatureMessage{msg1.Message.Messages[0], msg2.Message.Messages[0], msg3.Message.Messages[0]}
+
+	// Verify the reconstructed signature
+	expectedSig, err := types.ReconstructSignatures(map[types.OperatorID][]byte{1: msgs[0].PartialSignature, 3: msgs[2].PartialSignature})
+	if err != nil {
+		panic(err.Error())
+	}
```

Comment:
I am actually surprised that this doesn't err... 
you think we need this? anyhow if there is an `expectedError` then `expectedResult` isn't checked

Reply:
Indeed isn't checked and we don't need. It's only confusing so I'll drop.